content/applications/sales/sales/send_quotations/different_addresses.rst

```diff
@@ -1,75 +1,73 @@
-==========================================
-Deliver and invoice to different addresses
-==========================================
+==============================================
+Deliveries and invoices to different addresses
+==============================================
 
-With Odoo Sales, you can configure different addresses for delivery and invoicing. For some
-customers, it will be very practical to define specific billing and shipping addresses. Indeed, not
-everyone will have the same delivery location as the invoicing location.
+People and businesses often use separate addresses for billing (invoicing) and shipping (delivery)
+purposes. With the Odoo *Sales* app, contacts can have different specified addresses for delivery
+and invoicing.
 
-Activate the feature
-====================
+Settings
+========
 
-Go to :menuselection:`Sales --> Configuration --> Settings` and activate the **Customer Addresses**
-feature.
+To properly utilize multiple addresses in Odoo, go to :menuselection:`Sales app --> Configuration
+--> Settings` and scroll down to the :guilabel:`Quotations & Orders` heading. Then, check the box
+next to :guilabel:`Customer Addresses`, and click :guilabel:`Save`.
 
-.. image:: different_addresses/addresses_1.png
+.. image:: different_addresses/customer-addresses-setting.png
    :align: center
-   :class: img-thumbnail
-   :alt: How to enable different customer addresses on Odoo Sales?
+   :alt: Activate the Customer Addresses setting.
 
-Add addresses from a quotation
-==============================
+Configure the contact form
+==========================
 
-When you create a quotation, you must add a customer. This customer can be a company or a person
-with specific billing and shipping addresses already defined and registered in the system, or it can
-be a new customer. In this case, you have to **Create and edit** the contact form for your new
-customer and link it, if necessary, to a company. In this contact form, you will be able to add,
-delete and modify invoice and delivery addresses.
+To add multiple addresses to a contact, navigate to the :menuselection:`Contacts` app (or to
+:menuselection:`Sales --> Orders --> Customers`, and clear any default filters from the search
+bar), then click on the desired customer to open their contact form.
```

Comment:
You might want to break out the navigation in parentheses into a tip tag, just to make the copy a bit cleaner.